hibernate-core/src/main/java/org/hibernate/cfg/DependentSecondPass.java

```diff
@@ -0,0 +1,9 @@
+package org.hibernate.cfg;
+
+/**
+ * @author Matt Drees
+ */
+public interface DependentSecondPass extends SecondPass {
+
+	boolean dependentUpon(SecondPass secondPass);
```

Comment:
Thanks for the feedback!

Yeah, my sorting approach made more sense in the 4.x world than it does here. I left it mostly because I was lazy and wanted to get your input before fiddling more, but also partly because I think the sorting logic looks a little nicer using `DependentSecondPass` than `CopyIdentifierComponentSecondPass`.

I'll 'de-generify' this.